app/models/infra_conversion_job.rb

```diff
@@ -0,0 +1,160 @@
+class InfraConversionJob < Job
+  POLL_CONVERSION_INTERVAL = 60
+  POLL_CONVERSION_MAX = 24 * 60 # i.e. default 24 hour
+  POLL_POST_STAGE_MAX = 30 # i.e. default 30 minutes
+
+  def self.create_job(options)
+    # TODO: from settings/user plan settings
+    options[:conversion_polling_interval] = POLL_CONVERSION_INTERVAL
+    options[:poll_conversion_max] = POLL_CONVERSION_MAX
+    options[:poll_post_stage_max] = POLL_POST_STAGE_MAX
```

Comment:
Would rather these be `options[:poll_post_stage_max] ||= 24 * 60` so they can be modified by the caller